pallets/accounts/src/lib.rs

```diff
@@ -541,7 +559,7 @@ pub mod pallet {
 			let pub_key = &pub_key[12..];
 			ensure!(pub_key == sender.0, Error::<T>::VerificationFailed);
 
-			let data_amount: u128 =
+			let data_amount: u64 =
```

Comment:
hmm why 64 bits?

Reply:
Because i don't think that the amount of FRAG locked received from Ethereum will ever exceed 64 bit.

Reply:
we don't know, FRAG has 18 decimals, make it 128 at least please.

Reply:
Done!